### PR TITLE
Update runtime, youtube-dl and ffmpeg

### PR DESCRIPTION
**Pull request by Unrud**
The old version of youtube-dl doesn't work with some videos.

### CODE REVIEW

**Comment by TingPing** [comment]
You need to explicitly set `PYTHON=/usr/bin/python3` for `youtube-dl`.

**Comment by Unrud** [comment]
> You need to explicitly set `PYTHON=/usr/bin/python3` for `youtube-dl`:

Installing with **pip** works too, but I can change it back to **autotools** and set the environment variable instead.

**Comment by TingPing** [comment]
Ah didn't notice it was using pip. Either method is fine I suppose.

File: com.github.JannikHv.Gydl.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "app-id": "com.github.JannikHv.Gydl",
   "runtime": "org.gnome.Platform",
-  "runtime-version": "3.28",
+  "runtime-version": "3.30",
```
Comment (LeandroStanger) [inline com.github.JannikHv.Gydl.json]: Update runtime to 3.32

**Comment by nedrichards** [comment]
bot, build

**Comment by nedrichards** [comment]
bot, build

**Comment by nedrichards** [comment]
This doesn't appear to actually work at downloading videos any more.

**Comment by Unrud** [comment]
> This doesn't appear to actually work at downloading videos any more.

**youtube-dl** must be kept up to date. Somebody has to maintain this or it should be marked as end-of-life.

**Comment by AsavarTzeth** [comment]
I have been thinking about this a lot and personally I would just make it EOL. I don't have time for it and the application itself is practically unmaintained.

Besides there are now alternatives available on Flathub, like [Video Downloader](https://flathub.org/apps/details/com.github.unrud.VideoDownloader).

**Comment by nedrichards** [comment]
This now works in testing locally. Main issue - it needs python2 available, who knew! ;-) I'm going to push this shortly because people actually seem to care: https://discourse.flathub.org/t/gydl-doesnt-work/150

**Comment by Unrud** [comment]
> Main issue - it needs python2 available, who knew! ;-) 

It doesn't need Python 2. You switched to the wrong download https://github.com/flathub/com.github.JannikHv.Gydl/pull/9/commits/a0ad9ca278cc5c516ac8fe65f46f353ebcb9cf21#diff-dcfa46b52b2ca0388416883cb655daddR61.

The tarball [releases/…/youtube-dl-2020.02.16.tar.gz](https://github.com/ytdl-org/youtube-dl/releases/download/2020.02.16/youtube-dl-2020.02.16.tar.gz) contains the pre-built file **youtube-dl** with the interpreter set to ``#!/usr/bin/env python``. Therefore the build command ``make youtube-dl PYTHON=/usr/bin/python3`` does nothing.

You have to either switch the download to the source tarball [archive/2020.02.16.tar.gz](https://github.com/ytdl-org/youtube-dl/archive/2020.02.16.tar.gz) or run ``make clean`` before building.

**Pull request merged**